static/js/components/JobDetails.jsx

```diff
@@ -80,7 +80,10 @@ var AddForm = React.createClass({
         <label>Workload</label>
         <Input ref="workload" type="text" placeholder={workload_placeholder} />
         <label>Backend</label>
-        <Input ref="backend" type="text" placeholder={backend_placeholder} value="gearman" />
+        <Input type="select" ref="backend">
```

Comment:
can we set a default? I think it works with `<option value="gearman" selected="selected">...</option>`